taxi/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=20, unique=True)
+
+    def __str__(self):
+        return self.username
+
+    class Meta:
+        permissions = AbstractUser._meta.permissions
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)

```

Comment:
The `related_name` attribute is missing in your `ForeignKey` and `ManyToManyField`. According to the checklist, in the majority of cases, the name of the model in plural will be just right.